tests/Utils.hs

```diff
@@ -56,7 +56,7 @@ backCompatFlags :: [ExecutionFlag]
 backCompatFlags = [ FlagDisablePact40, FlagDisablePact43, FlagDisablePact44 ]
 
 nestedDefPactFlags :: [ExecutionFlag]
-nestedDefPactFlags = [ FlagDisablePact44 ]
+nestedDefPactFlags = []
```

Comment:
Why are we no longer disabling Pact 4.4 here?

Reply:
I was playing around with it, if the flag is not needed, we don't need to include it.